Source/WebCore/page/ChromeClient.h

```diff
@@ -668,13 +668,13 @@ class ChromeClient {
 
     virtual void textReplacementSessionUpdateStateForReplacementWithID(const UnifiedTextReplacement::SessionID&, UnifiedTextReplacement::ReplacementState, const UnifiedTextReplacement::ReplacementID&) { }
 
-    virtual void removeTextIndicatorStyleForID(const UnifiedTextReplacement::SessionID&) { }
+    virtual void removeTextAnimationForID(const UnifiedTextReplacement::SessionID&) { }
 
-    virtual void cleanUpTextStylesForSessionID(const UnifiedTextReplacement::SessionID&) { }
+    virtual void cleanUpTextAnimationsForSessionID(const UnifiedTextReplacement::SessionID&) { }
 
-    virtual void addSourceTextIndicatorStyle(const UnifiedTextReplacement::SessionID&, const CharacterRange&) { }
+    virtual void addSourceTextAnimation(const UnifiedTextReplacement::SessionID&, const CharacterRange&) { }
```

Comment:
`addWritingToolsSourceTextAnimationForSessionWithID`? (also, maybe label the CharacterRange parameter, since it's unclear what the parameter is